### DISCUSSION TITLE
Lower Minimum Mouse Sensitivity

### DISCUSSION THREAD

**Discussion by ReedClanton** [hyprwm/Hyprland#4528] [General]
# Description

For those with high DPI mice the lowest setting (-1) is still too high.

# Solution

Changing the sensitivity associated with -1, or 1, would break many people's environments, so perhaps the limits could be moved to accommodate people such as myself.

**Comment by kimjongbing** [comment]
i have this issue as well, was looking everywhere a solution. it is not niche

**Comment by ReedClanton** [comment]
An alternative to expanding the range from -1 to 1 would be to add an additional `gain` option. Setting it could increase the effective minimum and maximum sensitivity associated with -1 and 1.

**Comment by fufexan** [comment]
This is not possible. Hyprland simply passes the sensitivity to libinput, which can only accept values between -1 and 1. See https://wayland.freedesktop.org/libinput/doc/latest/api/group__config.html#ga1d9761bb99b445524fb2ea801f946ef1 for more info.

Perhaps you want to change the acceleration profile to flat.

**Reply by ReedClanton** [reply]
Well... that's a bummer. I already changed it to flat.
